test/data/fhir2cmumps-patient7.shex

```diff
@@ -0,0 +1,580 @@
+﻿PREFIX fhir: <http://hl7.org/fhir/> 
```

Comment:
It is not auto generated and it does need to be checked in.  One of the options on the roundtrip-translation script is the --backtarget option which needs to take a back translation file.  There are reasons why one might want one.